File: Data Structure and Algorithm/Assignment 01/Program 15/prog15.py
#Write a Python program to create an array containing six integers. Also, print all the members of the array.


import array as arr

a1 = arr.array('i')
n = 6
for i in range(n):
    val = int(input("Enter the element: "))
    a1.append(val)

for i in range(n):
    print(a1[i],end=' ')

File: Data Structure and Algorithm/Assignment 01/Program 15/prog15.c
//Write a Python program to create an array containing six integers. Also, print all the members of the array.

#include<stdio.h>
#include<stdlib.h>

int main(){
    int n=6;
    int *arr = (int*)malloc(n*sizeof(int));
    for (int i = 0; i < n; i++)
    {
        printf("Enter the number %d",i+1);
        scanf("%d",(arr+i));
    }
    for (int i = 0; i < n; i++)
    {
        printf("%d ",*(arr+i));
    }
    free(arr);
    return 0;
}